lib/appscale_tools.py

```diff
@@ -66,6 +69,22 @@ class AppScaleTools(object):
   ADMIN_CAPABILITIES = "upload_app"
 
 
+  # Command to cd into the AppScale repository.
+  CD_APPSCALE_REPO = "cd ~/appscale"
+
+
+  # Command to run the Bootstrap.
+  RUN_BOOTSTRAP_COMMAND = "bash bootstrap.sh"
+
+
+  # String of commands to cd into the ApppScale repository and run the Bootstrap
+  BOOTSTRAP_COMMAND = CD_APPSCALE_REPO + ";" + RUN_BOOTSTRAP_COMMAND
+
+
+  # Command to run the upgrade script from /appscale/scripts directory.
+  UPGRADE_SCRIPT = "python ~/appscale/scripts/datastore_upgrade.py"
```

Comment:
Instead of "~/appscale" can we use AppScale repo dir as a constant?
Create a new constant if there isn't one already.

That will help in making AppScale relocatable.